06/[Butter] Joshikousei no Mudazukai - 06 [1080p].ass

```diff
@@ -1,600 +1,566 @@
-Dialogue: 0,0:21:23.70,0:21:28.37,Default,,0,0,0,,will probably never forget the taste of the\NChoco Bat that I ate together with everyone.
-Dialogue: 0,0:21:30.79,0:21:32.00,Default,,0,0,0,,Big Sis.
-Dialogue: 0,0:21:32.00,0:21:35.45,Default,,0,0,0,,Please stop preserving anything\Nand everything in formalin!
-Dialogue: 0,0:21:36.50,0:21:38.12,Default,,0,0,0,,Probably for the rest of my life...
-Dialogue: 0,0:21:55.45,0:21:59.83,Default,,0,0,0,,Sawagashikute yakamashikute
-Dialogue: 0,0:22:00.12,0:22:03.70,Default,,0,0,0,,Demo itoshikute
-Dialogue: 0,0:22:04.04,0:22:07.91,Default,,0,0,0,,Ano hi no warai koe
-Dialogue: 0,0:22:07.91,0:22:11.91,Default,,0,0,0,,Maki modesenai story
-Dialogue: 0,0:22:11.91,0:22:15.79,Default,,0,0,0,,Seishun te yatsu wa zutto hikari to kage
-Dialogue: 0,0:22:15.79,0:22:17.83,Default,,0,0,0,,Kurayami no naka
-Dialogue: 0,0:22:17.83,0:22:19.87,Default,,0,0,0,,Kagayaku rettousei
-Dialogue: 0,0:22:20.08,0:22:23.58,Default,,0,0,0,,Furi kaereba kojiraseta
-Dialogue: 0,0:22:23.58,0:22:28.00,Default,,0,0,0,,Utsukushi sugiru hibi
-Dialogue: 0,0:22:28.54,0:22:37.58,Default,,0,0,0,,Itsumo haru wa fui ni toori sugita ato
-Dialogue: 0,0:22:37.58,0:22:44.66,Default,,0,0,0,,Kimi no kotoba mune ni reverb
-Dialogue: 0,0:22:44.66,0:22:53.16,Default,,0,0,0,,Itsuka haru ga zenbu tsure satte shimau
-Dialogue: 0,0:22:53.16,0:23:00.83,Default,,0,0,0,,Sono hi made wa baka de ii
-Dialogue: 0,0:23:01.16,0:23:05.41,Default,,0,0,0,,Kake nukeru kara
-Dialogue: 0,0:23:10.00,0:23:12.95,Default,,0,0,0,,This kind of movie really does\Nfeel heartwarming after all!
-Dialogue: 0,0:23:18.62,0:23:22.00,Default,,0,0,0,,Majo and Loli are probably enjoying\Nthe movie about now!
-Dialogue: 0,0:23:22.00,0:23:24.12,Default,,0,0,0,,Hmm? Robo, what're you drawing?
-Dialogue: 0,0:23:24.50,0:23:26.62,Default,,0,0,0,,I've tried to make one for Baka as well.
-Dialogue: 0,0:23:27.29,0:23:28.62,Default,,0,0,0,,A reincarnation Amida lottery.
-Dialogue: 0,0:23:28.62,0:23:31.12,Default,,0,0,0,,It's one of those about what you'll become\Nif you get reborn, right?
-Dialogue: 0,0:23:31.12,0:23:32.75,Default,,0,0,0,,Wow! Cool!
-Dialogue: 0,0:23:32.75,0:23:35.50,Default,,0,0,0,,{\an7}Reincarnation Amida lottery
-Dialogue: 0,0:23:32.75,0:23:34.16,Default,,0,0,0,,Then, I pick number five!
-Dialogue: 0,0:23:36.25,0:23:37.45,Default,,0,0,0,,{\an8}Face mite\N{\an8}Face mite\N{\an8}Face mite\N{\an8}Face mite
-Dialogue: 0,0:23:39.08,0:23:40.33,Default,,0,0,0,,Face mite...
+Dialogue: 0,0:00:00.00,0:00:00.00,Default,chapter,0,0,0,,{Intro}Intro
+Dialogue: 0,0:01:28.98,0:01:28.98,Default,chapter,0,0,0,,{OP}OP
+Dialogue: 0,0:02:59.03,0:02:59.03,Default,chapter,0,0,0,,{Part A}Part A
+Dialogue: 0,0:11:49.10,0:11:49.10,Default,chapter,0,0,0,,{Part B}Part B
+Dialogue: 0,0:21:38.06,0:21:38.06,Default,chapter,0,0,0,,{ED}ED
+Dialogue: 0,0:23:08.03,0:23:08.03,Default,chapter,0,0,0,,{Outro}Outro
+Dialogue: 0,0:00:00.58,0:00:03.10,Default,,0,0,0,,Hey, can I say something amazing right now?
```

Comment:
lead in too low
![image](https://user-images.githubusercontent.com/60306998/97098982-536f0180-16a9-11eb-8158-620509bce8b8.png)